### DISCUSSION TITLE
Opening / closing tag non corrispondenti (articolo E-Invoice XML customization)

### DISCUSSION THREAD

**Discussion by morinim** [fattureincloud/api#400] [General]
Nell'articolo [E-Invoice XML customization](https://developers.fattureincloud.it/docs/guides/e-invoice-xml-customisation) bisognerebbe correggere l'esempio nella sezione [Items List](https://developers.fattureincloud.it/docs/guides/e-invoice-xml-customisation/#ramen-items-list) cambiando il riquadro contenente l'XML da:

---
This is the final XML that we want to obtain:
```
<FatturaElettronicaBody>
  <DatiBeniServizi>
    <DettaglioLinee>
      <AltriDatiGestionali>
        <TipoDato>TIPO_DATO</TipoDato>
      </AltriDatiGestionali>
    </DatiRicezione>
  </DatiGenerali>
</FatturaElettronicaBody>
```
---

a

---
This is the final XML that we want to obtain:

```
<FatturaElettronicaBody>
  <DatiBeniServizi>
    <DettaglioLinee>
      <AltriDatiGestionali>
        <TipoDato>TIPO_DATO</TipoDato>
      </AltriDatiGestionali>
    </DettaglioLinee>
  </DatiBeniServizi>
</FatturaElettronicaBody>
```
---

**Comment by emajo** [comment]
Ciao @morinim, grazie mille per la segnalazione, abbiamo provveduto a sistemare.
Emanuele.
